diff --git a/smtp/message.ts b/smtp/message.ts
index b1fc755b089edcde4a66b522a401764c2776db1c..06ef5e7a85fc8435f57c90137f403f4a6c8d7717 100644
--- a/smtp/message.ts
+++ b/smtp/message.ts
@@ -1,4 +1,5 @@
-import fs, { PathLike } from 'fs';
+import { PathLike } from 'fs';
+import * as fs from 'fs';
 import { hostname } from 'os';
 import { Stream } from 'stream';
 import type { Readable } from 'stream';
